core/include/webview/detail/engine_base.hh

```diff
@@ -134,7 +134,9 @@ window.__webview__.onUnbind(" +
   noresult set_title(const std::string &title) { return set_title_impl(title); }
 
   noresult set_size(int width, int height, webview_hint_t hints) {
-    return set_size_impl(width, height, hints);
+    set_size_impl(width, height, hints);
+    m_is_size_set = true;
+    return noresult{};
```

Comment:
Please return the result of `set_size_impl()`.

Reply:
See latest commit.

Would you please re-run the 3 timed out CI checks.